components/dashboard/src/data/workspaces/delete-inactive-workspaces-mutation.ts

```diff
@@ -6,17 +6,14 @@
 
 import { useMutation, useQueryClient } from "@tanstack/react-query";
 import { workspacesService } from "../../service/public-api";
-import { getGitpodService } from "../../service/service";
 import { getListWorkspacesQueryKey, ListWorkspacesQueryResult } from "./list-workspaces-query";
 import { useCurrentOrg } from "../organizations/orgs-query";
-import { useFeatureFlag } from "../featureflag-query";
 
 type DeleteInactiveWorkspacesArgs = {
     workspaceIds: string[];
 };
 export const useDeleteInactiveWorkspacesMutation = () => {
     const queryClient = useQueryClient();
-    const usePublicApiWorkspacesService = useFeatureFlag("publicApiExperimentalWorkspaceService");
```

Comment:
Random side note: This feature flag is off for Dedicated. Intentional?

Reply:
Probably not